perun-openapi/openapi.yml

```
@@ -2622,6 +2639,14 @@ components:
       in: query
       required: true
 
+    consentHubId:
+      name: consentHubId
```

```suggestion
      name: consentHub
```
This is just a nit-pick and last one, I promise :D I think this would be better because that's how we name other "id" parameters, and this consistency could prevent future minor mistakes.